user-guide/Advanced_Modules/DOM/DOM_examples/DOM_BulkProcessing_Example.md

```diff
@@ -0,0 +1,100 @@
+
+For the `DomInstances` that failed their name and the issue will be logged in this example.
+
+  ```csharp
+  // Retrieve the DomInstances that need processing
+  var domInstances = domHelper.DomInstances.Read(filter);
+
+  // Update the field value on those DomInstances
+  foreach(var domInstance in domInstances)
+  {
+      domInstance.AddOrUpdateFieldValue<int>(sectionDefinitionId, fieldDescriptorId, newType);
+  }
+
+  // Save them to the DB.
+  var updateResult = domHelper.DomInstances.CreateOrUpdate(domInstances);
+
+  // Log what items were successfully removed
+  Log($"The new type was successfully set on {updateResult.SuccessfulItems.Count}");
+
+  // Log the some updates went wrong. Per DomInstance check how what issue occurred.
+  var incorrectInstances = updateResult.TraceDataPerItem.Where(x => !x.Value.HasSucceeded()).ToList();
+  if (incorrectInstances.Count > 0)
+  {
+      var issues = incorrectInstances.Select(x => $"{domInstances.FirstOrDefault(y => x.Equals(y.ID))?.Name}: {x.Value}");
+      Log($"The type couldn't be updated for {incorrectInstances.Count} items: {string.Join(Environment.NewLine, issues)}");
+  }
+  ```
+
+## Create multiple DomInstances
+
+In this example two `DomInstances` get created. The second `DomInstance` will be created using a `FieldDescriptor` that no longer is supported for initial state it gets created in.
+
+In this example the `TraceData` will be logged. Depending on the issue that occurs, the ID of the `DomInstance` might be included in the error.
+
+  ```csharp
+  // Instantiate a new DomInstance & assign 'Hello' as a value for a correct FieldDescriptor.
+  var domInstanceOne = new DomInstance() { DomDefinitionId = basicDomDefinitionId };
+  domInstanceOne.AddOrUpdateFieldValue<string>(sectionDefinitionId, fieldDescriptorId, "Hello");
+
+  // Instantiate a new DomInstance & assign 'World' as a value for a FieldDescriptor no longer supported for the initial state.
+  var domInstanceTwo = new DomInstance() { DomDefinitionId = advancedDomDefinitionId };
+  domInstanceTwo.AddOrUpdateFieldValue<string>(sectionDefinitionId, incorrectFieldDescriptorId, "World");
+
+  // Save them to the DB
+  var domInstances = new List<DomInstance> { domInstanceOne, domInstanceTwo };
+  domHelper.DomInstances.CreateOrUpdate(domInstances);
+
+  // Log if an issue occurred
+  var traceData = domHelper.DomInstances.GetTraceDataLastCall();
+  if (!traceData.HasSucceeded())
+  {
+      Log($"Some of the instances couldn't be created: {traceData}");
+  }
+  ```
+
+## Remove multiple DomInstances
+
+In this example some `DomInstances` get filter out to be removed.
+
+If, for example, the user executing this example isn't allowed to remove them, it will be logged for each `DomInstance`.
+In case the `Read` or `Delete` call was unsuccessful because of a database issue, a `CrudFailedException` will be thrown. Details of the issue will available in the `TraceData` of that exception.
```

Comment:
```suggestion
In case the `Read` or `Delete` call is unsuccessful because of a database issue, a `CrudFailedException` will be thrown. Details of the issue will be available in the `TraceData` of that exception.
```